src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -217,6 +226,8 @@ def load_arguments(self, _):
         c.argument('load_balancer_outbound_ip_prefixes', validator=validate_load_balancer_outbound_ip_prefixes)
         c.argument('load_balancer_outbound_ports', type=int, validator=validate_load_balancer_outbound_ports)
         c.argument('load_balancer_idle_timeout', type=int, validator=validate_load_balancer_idle_timeout)
+        c.argument('load_balancer_backend_pool_type', arg_type=get_enum_type(backend_pool_types),
+                   validator=validate_load_balancer_backend_pool_type)
```

Comment:
After defined with `arg_type=get_enum_type(backend_pool_types)`, it is not necessary to validate the input manually.
```suggestion
        c.argument('load_balancer_backend_pool_type', arg_type=get_enum_type(backend_pool_types))
```

Reply:
removed